src/test/java/de/medizininformatikinitiative/flare/service/StructuredQueryServiceIT.java

```diff
@@ -134,12 +136,11 @@ void execute_Criterion() {
         assertThat(result).isOne();
     }
 
-    @Test
+    // Commented out - as current ontology has no lab values in mapping
+    //@Test
```

Comment:
Please remove the test. It's ok to only test stuff, we have an onto for. But it's not so nice to leave behind commented code.